src/editor/CodeHintList.js

```diff
@@ -503,7 +505,16 @@ define(function (require, exports, module) {
                                 "width": hintPos.width + "px"});
         }
     };
-
+    /**
+     * Calls the move up keybind to move hint suggestion selector
+     *
+     * @param {KeyBoardEvent} keyEvent
+     */
+    CodeHintList.prototype.callMoveUp = function (event) {
+        delete event.type;
```

Comment:
I think you can also create a new event and use the old event as parameters 

```
var newEvent = new Event("keydown", event);
```

Not sure if the original `event.type` overrides the other new type though.